gnovm/cmd/gno/main.go

```diff
@@ -4,16 +4,17 @@ import (
 	"context"
 	"os"
 
+	"github.com/gnolang/gno/gnovm/cmd/gno/internal/pkgdownload"
 	"github.com/gnolang/gno/tm2/pkg/commands"
 )
 
 func main() {
-	cmd := newGnocliCmd(commands.NewDefaultIO())
+	cmd := newGnocliCmd(commands.NewDefaultIO(), nil)
```

Comment:
Because this is only necessary in tests, I think we should simply have something like `var testPackageFetcher` which is set up at test init, which we switch to using in `mod.go` if available.

Reply:
16f14308d21bc66eb8f1e173e62948a18361d600